### PR TITLE
Issue421 durack1 add seawater packages

### PR DESCRIPTION
**Pull request by durack1**
@doutriaux1 take a look at this in response to #421

### CODE REVIEW

**Comment by durack1** [comment]
@doutriaux1, yep also seems like we've got these in the build queue too:
`[INFO] These are the packages that we will try to build: basemap;CDAT;CURL;Cython;FFMPEG;g2clib;gdal;GEOS;**gsw**;gui_support;h5py;HDF5;ZMQ;pyzmq;tornado;IPYTHON;jasper;jpeg;LAPACK;lepl;libcf;Matplotlib;SIX;DATEUTILS;PYPARSING;Mpi4py;PYOPENSSL;setuptools;MyProxyClient;NetCDF;NetCDFPLUS;Numexpr;NUMPY;MPI;OSMesa;Python;**seawater**;VTK;pkgconfig;png;pyspharm;PyTables;readline;R;scikits;SCIPY;pip;Shapely;pygments;markupsafe;jinja2;docutils;Sphinx;freetype;zlib;libXML2;libXSLT;png;pixman;fontconfig;tiff;Curses;CMOR;udunits2;uuid;UVCMETRICS;vistrails;YASM;PYLIBXML2;CDATLogger;pyclimate;scientificpython;windspharm;eof2;eofs;windfield`

**Comment by doutriaux1** [comment]
@durack1 you need to make them dependent on python at least

**Comment by durack1** [comment]
@doutriaux1 isn't that achieved in the gsw/seawater_deps.cmake files `set(gsw ${gsw_pkg} ${python_pkg})`

**Comment by durack1** [comment]
@doutriaux1 ok these both need numpy
gsw: https://github.com/ocefpaf/python-gsw/blob/master/setup.py#L81-82
seawater: https://github.com/ocefpaf/python-seawater/blob/master/setup.py#L60-61

**Comment by durack1** [comment]
@doutriaux1 ok this one should be good to go - I fixed the dependencies

File: CMake/cdat_modules/gsw_pkg.cmake [hunk 1]
```diff
@@ -0,0 +1,18 @@
+set( GSW_MAJOR 3 )
+set( GSW_MINOR 0 )
+set( GSW_PATCH 3 )
+set( GSW_VERSION ${GSW_MAJOR}.${GSW_MINOR}.${GSW_PATCH} )
+set( GSW_URL ${LLNL_URL} )
+set( GSW_GZ python-gsw-${SEAWATER_VERSION}.tar.gz )
```
Comment (aashish24) [inline CMake/cdat_modules/gsw_pkg.cmake]: This won't work. SEAWATER_VERSION is not set globally or in parent scope.

**Comment by aashish24** [comment]
@durack1 please see my comment above. This branch needs some fix.

**Comment by durack1** [comment]
@aashish24 yep thanks for that, it should now be fixed..

**Comment by aashish24** [comment]
thanks. LGTM :+1:

**Pull request merged**